bin/key_server.js

```diff
@@ -110,6 +110,7 @@ function main() {
         customs.close()
         mailer.stop()
         database.close()
+        process.exit() //XXX: because of openid dep ಠ_ಠ
```

Comment:
the openid module doesn't close its connections, so this function was no longer terminating the process in a timely manner, causing test files to hang at the end